pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -532,6 +538,20 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 		dexCfg = render.NewDexRelyingPartyConfig(authentication, dexTLSSecret, dexSecret, r.clusterDomain)
 	}
 
+	// Cloud modifications
+	kbCm := &corev1.ConfigMap{}
+	key := types.NamespacedName{Name: "cloud-kibana-config", Namespace: rmeta.OperatorNamespace()}
+	if err = r.client.Get(ctx, key, kbCm); err != nil {
+		if !errors.IsNotFound(err) {
+			return reconcile.Result{}, fmt.Errorf("Failed to read cloud-kibana-config ConfigMap: %s", err.Error())
+		}
+	} else {
+		if err = json.Unmarshal([]byte(kbCm.Data["config"]), &render.CloudKibanaConfig); err != nil {
```

Comment:
FYI - ISTR that configmaps have a field `.BinaryData` for accessing data in byte form, removing the need to cast the string as a byte array.

But I also STR that there are cases where that won't be filled in, so it's only worth switching to it if you can quickly verify it works as expected. So fine leaving this change out.

Reply:
It looks like the `.BinaryData` is actually a whole different field ... I thought the same as you until I took a look at the ConfigMap. 
```
    // BinaryData contains the binary data.
    // Each key must consist of alphanumeric characters, '-', '_' or '.'.
    // BinaryData can contain byte sequences that are not in the UTF-8 range.
    // The keys stored in BinaryData must not overlap with the ones in
    // the Data field, this is enforced during validation process.
    // Using this field will require 1.10+ apiserver and
    // kubelet.
    // +optional
    BinaryData map[string][]byte `json:"binaryData,omitempty" protobuf:"bytes,3,rep,name=binaryData"`
```